src/DependencyInjection/SyliusInvoicingExtension.php

```diff
@@ -20,4 +21,23 @@ public function load(array $config, ContainerBuilder $container): void
 
         $loader->load('services.xml');
     }
+
+    public function prepend(ContainerBuilder $container): void
+    {
+        if (!$container->hasExtension('doctrine_migrations') || !$container->hasExtension('sylius_core')) {
+            return;
+        }
+
+        $container->prependExtensionConfig('doctrine_migrations', [
+            'migrations_paths' => [
```

Comment:
Will it be required to do it like that, or can we move these configurations to YAML?